src/SFA.DAS.AdminService.Web/ModelBinders/SuppressBindingErrorsModelBinderProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Binders;
+using SFA.DAS.AdminService.Web.Attributes;
+
+namespace SFA.DAS.AdminService.Web.ModelBinders
```

Comment:
Might want to consider putting this into SFA.DAS.AdminService.Common at a later date